app/services/recaptcha_validator.rb

```diff
@@ -0,0 +1,56 @@
+class RecaptchaValidator
+  VERIFICATION_ENDPOINT = 'https://www.google.com/recaptcha/api/siteverify'.freeze
+
+  class ValidationError < StandardError; end
+
+  attr_reader :score_threshold, :analytics
+
+  def initialize(score_threshold: 0.0, analytics: nil)
+    @score_threshold = score_threshold
+    @analytics = analytics
+  end
+
+  def exempt?
+    !score_threshold.positive?
+  end
+
+  def valid?(recaptcha_token)
+    return true if exempt?
+
+    response = faraday.post(
```

Comment:
could we add the service to the faraday instrumentation ([docs](https://github.com/18F/identity-idp/blob/e6a56e86bf8ed1252e83e253b39220d7ba409913/docs/backend.md#networking))?

Reply:
Ah, I [added the instrumentation](https://github.com/18F/identity-idp/blob/655e2560704b15ffe83ec23be2d35156a0b12a2d/app/services/recaptcha_validator.rb#L44), but I missed the `service_name` bit of it. I'll add.

Reply:
Updated in 9cecd08f4.